src/middleware/withSSRLogger.ts

```diff
@@ -0,0 +1,51 @@
+import { createLogger } from "@fewlines/fwl-logging";
+import { GetServerSideProps } from "next";
+import { GetServerSidePropsResult } from "next";
+
+export function withSSRLogger(
+  getServerSideProps: GetServerSideProps,
+): GetServerSideProps {
+  const logger = createLogger("connect-account", "json").withMeta({
+    process: "getServerSideProps",
+  });
+
+  const processTimeStart = process.hrtime.bigint();
+
+  return async (context) => {
+    return getServerSideProps(context)
+      .then(
+        (
+          response: GetServerSidePropsResult<{
```

Comment:
is this a `response` or is it `props`?

Reply:
It is `props`, but I went with this as it is the response from the promise above all imo (I was a bit confused about the semantic about this one). It is still working, but I can change the naming if you want.

Reply:
well, seeing the type, what about `serverSidePropsResult`? A bit verbose but at least, we know 😁